pkg/vpcmodel/grouping.go

```diff
@@ -67,44 +67,44 @@ func newGroupingConnections() *groupingConnections {
 
 func newGroupConnLines(c *VPCConfig, v *VPCConnectivity,
 	grouping bool) (res *GroupConnLines, err error) {
+	initCacheGrouped := initCacheGroupedElem()
 	res = &GroupConnLines{config: c, nodesConn: v,
-		srcToDst:                 newGroupingConnections(),
-		dstToSrc:                 newGroupingConnections(),
-		groupedEndpointsElemsMap: map[string]*groupedEndpointsElems{},
-		groupedExternalNodesMap:  map[string]*groupedExternalNodes{}}
+		srcToDst:     newGroupingConnections(),
+		dstToSrc:     newGroupingConnections(),
+		cacheGrouped: initCacheGrouped}
 	err = res.computeGrouping(true, grouping)
 	return res, err
 }
 
 func newGroupConnLinesSubnetConnectivity(c *VPCConfig, s *VPCsubnetConnectivity,
 	grouping bool) (res *GroupConnLines, err error) {
+	initCacheGrouped := initCacheGroupedElem()
```

Comment:
same here and in the other functions below

Reply:
right